src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -100,6 +100,8 @@ public IReadOnlyNetworkPeerCollection ConnectedNodes
 
         private readonly Dictionary<NetworkPeer, PerformanceSnapshot> downloads = new Dictionary<NetworkPeer, PerformanceSnapshot>();
 
+        private NetworkPeerServices discoveredNodeRequiredService = NetworkPeerServices.Network;
```

Comment:
Ah ok yes this is about what we are looking for when discovering peers and not what are our capabilities.
yes this should probably be Network in that case.